projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_dag.py

```diff
@@ -12,12 +12,56 @@
 from typing import List
 
 from taurus_datajob_api import ApiException
+from vdk.internal.core import errors
 from vdk.plugin.meta_jobs.cached_data_job_executor import TrackingDataJobExecutor
 from vdk.plugin.meta_jobs.meta import TrackableJob
 from vdk.plugin.meta_jobs.remote_data_job_executor import RemoteDataJobExecutor
 from vdk.plugin.meta_jobs.time_based_queue import TimeBasedQueue
 
 log = logging.getLogger(__name__)
+max_starting_jobs = 15
+
+
+def validate_job_limit(jobs: List[Dict]):
```

Comment:
Could you provide code docs about the purpose of `validate_job_limit`, for example explaining expected behaviour towards depends_on graph elements

Reply:
Not relevant anymore.